sklearn/neural_network/multilayer_perceptron.py

```diff
@@ -710,6 +737,9 @@ class MLPClassifier(BaseMultilayerPerceptron, ClassifierMixin):
         - 'relu', the rectified linear unit function,
           returns f(x) = max(0, x)
 
+        - can be a list of function for different activation function
```

Comment:
You should modify the type description right after the parameter name

Reply:
fixed.